src/interface_mssql.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
```

Comment:
drop that line; this file is not executable (nor should it be); same goes for the other non-executable files here.

a module docstring would be nice.